src/Spryker/Shared/AppPayment/Transfer/app_payment.transfer.xml

```diff
@@ -416,17 +418,32 @@
     <transfer name="PaymentCreated" strict="true">
         <property name="entityReference" type="string"/>
         <property name="paymentReference" type="string"/>
-        <property name="details" type="string"/>
+        <property name="details" type="string" description="A JSON representation of payment details such as PSP specific data and payment status history data"/>
         <property name="messageAttributes" type="MessageAttributes"/>
     </transfer>
 
     <transfer name="PaymentUpdated" strict="true">
         <property name="entityReference" type="string"/>
         <property name="paymentReference" type="string"/>
-        <property name="details" type="string"/>
+        <property name="details" type="string" description="A JSON representation of payment details such as PSP specific data and payment status history data"/>
         <property name="messageAttributes" type="MessageAttributes"/>
     </transfer>
 
+    <transfer name="PaymentStatusHistory" strict="true">
+        <property name="status" type="string"/>
+        <property name="createdAt" type="string"/>
+    </transfer>
+
+    <transfer name="PaymentStatusHistoryCollection" strict="true">
+        <property name="paymentStatusHistory" type="PaymentStatusHistory[]" singular="paymentStatusHistory"/>
```

Comment:
```suggestion
        <property name="paymentStatusHistoryItems" type="PaymentStatusHistory[]" singular="paymentStatusHistoryItem"/>
```

Reply:
I was also thinking about a different name here. "Item" reminds me too much of an order item. How about using just `PaymentStatus` and `PaymentStatusHistory`? The latter is the collection and the first is an "item" in the history.

I will go for it.